engine/src/main/java/org/terasology/rendering/shader/ShaderParametersShadowMap.java

```diff
@@ -22,10 +22,8 @@
  *
  */
 public class ShaderParametersShadowMap extends ShaderParametersBase {
-
```

Comment:
You would have reduced the number of files to review by at least 10% if you didn't touch these lines! :slightly_smiling_face: